docs/requirements/channel-upgradability-requirements.md

```diff
@@ -0,0 +1,178 @@
+# Business requirements
+
+Rather than create a new channel to expand upon the capabilities of an existing channel, channel upgradability enables new features and capabilities to be added to existing channels.
+
+## Problem
+
+Currently, once a channel is opened and the channel handshake is complete, you cannot change or renegociate the semantics of that channel. This means that if you wanted to make a change to a channel affecting the semantics on both channel ends, you would need to open a new channel, meaning all previous state in the prior channel would be lost. This is particularly important for channels using the ICS 20 (fungible token transfer) application module because tokens are not fungible between channels.
+
+Upgrading a channel enables upgrading the application module claiming the channel, where the upgrade requires a new packet data structure or adding a middleware at both channel ends. Currently it is possible to make changes to one end of a channel that does not require the counterparty to make changes, for example adding rate limiting middleware. Channel upgradability is solving the problem when changes need to be agreed upon on both sides of the channel.
+
+## Objectives
+
+To enable existing channels to upgrade the application module claiming the channel or add middleware to both ends of an existing channel whilst retaining the state of the channel.
+
+## Scope
+
+A new `ChannelEnd` interface is defined after a channel upgrade, the scope of these upgrades is detailed in the table below.
+
+| Features  | Release |
+| --------- | ------- |
+| Performing a channel upgrade results in an application module changing from v1 to v2, claiming the same `channelID` and `portID` | v8.1 |
```

Comment:
app module version going from v1 to v2 is not always true, slightly unclear whether its meant that this happens every time or is just a possibility that may happen on a particular upgrade

Reply:
It's an example for an upgrade that may happen.

Reply:
ah, fair enough! thanks